db/migrate/20220325110902_add_tsvector_columns.rb

```diff
@@ -0,0 +1,27 @@
+class AddTsvectorColumns < ActiveRecord::Migration[6.1]
+  def up
+    add_column :customers, :tsv, :tsvector
+    add_index :customers, :tsv, using: 'gin'
+
+    execute <<-SQL
+      CREATE TRIGGER tsvectorupdate BEFORE INSERT OR UPDATE
```

Comment:
For whatever reason, this comes as a surprise. I honestly thought that Postgres would automatically update the index. Of course, the reason I asked you to do this was because I (clearly) don't know how this is supposed to work.

So: is it 100% confirmed that you need to create a trigger and regenerate the tsvector manually?